sigh/lib/sigh/download_all.rb

```diff
@@ -55,8 +55,6 @@ def download_all(download_xcode_profiles: false)
         ]
       end
 
-      # Filtering on 'profileType' seems to be undocumented as of 2020-07-30
-      # but works on both web session and official API
```

Comment:
It is actually mentioned in the documentation at this moment (2022-05-22)
https://developer.apple.com/documentation/appstoreconnectapi/list_and_download_profiles#query-parameters